doc/topics/windows/multi-minion.rst

```diff
@@ -0,0 +1,164 @@
+=============================
+Multi-Minion Setup on Windows
+=============================
+
+There may be a scenario where having a minion running in the context of the
+current, logged-in user would be useful. For example, the normal minion running
+under the service account would perform machine-wide, administrative tasks. The
+minion running under the user context could be launched when the user logs in
+and would be able to perform configuration tasks as if it were the user itself.
+
+The steps required to do this are as follows:
+
+1. Create new root_dir
+2. Set root_dir permissions
+3. Create directory structure
+4. Write minion config
+
+We will now go through each of these steps in detail.
+
+.. note::
+
+    We have created a powershell script that will configure an additional minion
+    on the system for you. It can be found in the root of the Salt installation.
+    The script is named ``multi-minion.ps1``. You can get help on how to use the
+    script by running the following in a PowerShell prompt:
```

Comment:
**Suggestion (style, blocking):** Related to my previous comment, this needs to change to the second person. But I'm unclear if "we" here means the Salt team? If so, my suggested fix is:

```
The Salt Project has created a powershell script that will configure an additional minion on the system for you...
```